packages/js/core/src/algorithms/get-implementations.ts

```
@@ -24,9 +25,19 @@ export const getImplementations = Tracer.traceFunc(
       wrapperInterfaceUri: Uri
     ) => {
       for (const interfaceImplementations of implementationsArray) {
-        const fullyResolvedUri = redirects
-          ? applyRedirects(interfaceImplementations.interface, redirects)
-          : interfaceImplementations.interface;
+        let fullyResolvedUri: Uri;
+        if (redirects) {
+          const redirectsResult = applyRedirects(
+            interfaceImplementations.interface,
+            redirects
+          );
+          if (!redirectsResult.ok) {
+            continue;
```

Why are we ignoring the error here instead of returning it?

---

I was just trying not to change the behavior.

---

Isn't the previous implementation to throw an error here, but the current to silence it?

---

if so, I made a mistake!

---

You're right. The old implementation looked like this:
```
  const fullyResolvedUri = redirects
    ? applyRedirects(interfaceImplementations.interface, redirects)
    : interfaceImplementations.interface;
```
And applyRedirects threw errors. Good catch!!!

---

I fixed it in this PR: https://github.com/polywrap/toolchain/pull/1288

---

Great!